src/backend/distributed/cdc/Makefile

```diff
@@ -0,0 +1,26 @@
+ifndef DECODER
+	DECODER = pgoutput
+endif
+
+MODULE = citus_$(DECODER)
```

Comment:
not sure why, but I think this should be MODULE_big, otherwise I get
```
/usr/bin/install -c -m 755  citus_pgoutput.so '/home/marco/pgsql/15/share/contrib/citus_decoders/pgoutput.so'
/usr/bin/install: cannot stat 'citus_pgoutput.so': No such file or directory
```